BrowseRouter/BrowserService.cs

```diff
@@ -38,7 +38,7 @@ public async Task LaunchAsync(string url, string windowTitle)
       Log.Write($"Launching {path} with args \"{args} {uri.OriginalString}\"");
 
       string name = GetAppName(path);
-      await notifier.NotifyAsync($"Opening {name}", $"URL: {url}");
```

Comment:
The `await` keyword must remain. Removing it does let the current method continue as you intended, but then the program will exit before `NotifyAsync` has completed. On Windows 10, this will result in the tray icon remaining in the tray after the app has exited until the mouse moves overs over it, and then Windows removing the icon it doesn't respond to a window message. This is the issue adding the delay sought to fixed in the first place.

Moreover, in C# the correct way let the current method continue is by wrapping a call to `NotifyAsync` in `Task.Run`. As it currently is, any exception that `NotifyAsync` throws will be silently eaten, and on single-threaded environments this can deadlock (think Blazor).